pkg/networkutils/network.go

```diff
@@ -110,32 +109,51 @@ func (os *linuxNetwork) SetupHostNetwork(vpcCIDR *net.IPNet, primaryAddr *net.IP
 	// Only include the rule if SNAT is not being handled by an external NAT gateway and needs to be
 	// handled on-node.
 	if !externalSNAT {
-		err = os.netLink.RuleAdd(hostRule)
+		err = n.netLink.RuleAdd(hostRule)
 		if err != nil {
 			log.Errorf("Failed to add host IP rule: %v", err)
 			return errors.Wrapf(err, "host network setup: failed to add host rule")
 		}
 	}
 
+	// Configure the kernel's reverse path filter check on eth0 for "loose" filtering.  This is required because
+	// - NodePorts are exposed on eth0
+	// - The kernel's RPF check happens after incoming packets to NodePorts are DNATted to the pod IP.
+	// - For pods assigned to secondary ENIs, the routing table includes source-based routing.  When the kernel does
+	//   the RPF check, it looks up the route using the pod IP as the source.
+	// - Thus, it finds the source-based route that leaves via the secondary ENI.
+	// - In "strict" mode, the RPF check fails because the return path uses a different interface to the incoming
+	//   packet.  In "loose" mode, the check passes because some route was found.
+	f, err := os.OpenFile("/proc/sys/net/ipv4/conf/eth0/rp_filter", os.O_WRONLY, 0644)
```

Comment:
Is there any API for this? (instead of directly manipulate /proc/....?  ) 
Can you also create a function for this?